BedrockServer.cpp

```diff
@@ -1637,3 +1642,18 @@ SData BedrockServer::_generateCrashMessage(const BedrockCommand* command) {
     message.content = subMessage.serialize();
     return message;
 }
+
+void BedrockServer::onNodeLogin(SQLiteNode::Peer* peer)
+{
+    shared_lock<decltype(_crashCommandMutex)> lock(_crashCommandMutex);
+    for (const auto& p : _crashCommands) {
+        SINFO("Sending crash command " << p.first << " to node " << peer->name << " on login");
```

Comment:
Can we make this an alert to match the log level of the original broadcast messages?